src/main/java/org/example/controllers/JobRoleController.java

```diff
@@ -0,0 +1,33 @@
+package org.example.controllers;
+
+import io.swagger.annotations.Api;
+import org.example.services.JobRoleService;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+@Api("Kainos Job Application - Job Roles API")
+@Path("/api/")
```

Comment:
Question - have you tested this with swagger?  I notice this class level path ends in a '/' and the method level path starts with a '/'.  If all is ok via swagger, then no issue